content/en/docs/usage/kubectl-plugin.md

```diff
@@ -79,4 +81,22 @@ The default output will be printed to stdout in YAML format. One can use -o opti
 For example this will output `cert.yaml` in the latest API version:
 ```console
 kubectl cert-manager convert -f cert.yaml
+```
+
+### Create
+`kubectl cert-manager create` can be used to create cert-manager resources manually. Subcommands are available
+to create different resources:
+#### Certificaterequest
```

Comment:
```suggestion
#### certificaterequest
```

I think we should either match the case of the subcommand or match the case of the resource, e.g. `CertificateRequest`.